style.css

```diff
@@ -0,0 +1,282 @@
+html {
+  font-size: 62.5%;
+  text-rendering: optimizeLegibility;
+  -moz-osx-font-smoothing: grayscale;
+  font-smoothing: antialiased;
+  -webkit-font-smoothing: antialiased;
+  text-shadow: rgba(0, 0, 0, 0.01) 0 0 1px;
+}
+
+body {
+  font-size: 1.4rem;
+  font-family: sans-serif;
+  margin: 0;
+  padding: 4rem 0 0;
+}
+
+@media screen and (min-width: 737px) {
+  body {
+    padding-top: 0;
+  }
+}
+
+div.wrapper {
+}
+
+@media screen and (min-width: 737px) {
+  div.wrapper {
+    display: flex;
+    flex-direction: row;
+  }
+}
+
+.sidebar label {
+  height: 4rem;
+  width: 100%;
+  color: #fff;
+  background-color: #225ba0;
+  display: inline-block;
+  text-align: left;
+  padding: 0 1rem;
+  font-size: 2rem;
+  line-height: 4.4rem;
+  z-index: 10;
+  position: fixed;
+  top: 0;
+}
+
+aside {
+  padding: 1rem;
+  background-color: #225ba0;
+  box-sizing: border-box;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  overflow: visible;
+}
+
+aside button {
+  top: 0;
+  left: 100%;
+  background-color: #225ba0;
+  color: #fff;
+  position: absolute;
+  height: 4rem;
+  width: 4rem;
+  border: 0;
+}
+
+aside img {
+  max-width: 15rem;
+}
+
+aside ul.categories {
+  margin: 0 0 2rem;
+  padding: 0;
+  list-style-type: none;
+  display: none;
+  flex-wrap: wrap;
+}
+
+aside ul.categories li {
+  margin-bottom: 1rem;
+  margin-right: 1rem;
+}
+
+aside ul.categories li a {
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-size: 1.2rem;
+  text-align: center;
+  background: #fff;
+  padding: 1rem 2rem 0.8rem;
+  width: 100%;
+  text-decoration: none;
+  box-sizing: border-box;
+  display: block;
+  border-radius: 4px;
+  box-shadow: 0px 1px 1px #888;
+}
```

Comment:
Worth using SASS here?

Reply:
Yeah - I didn't really want to have to build a whole SASS thing, but now I've written it I think it's worth the investment